README.md

```diff
@@ -57,6 +58,22 @@ client = Bitreserve::Client.new
 puts client.all_tickers
 ```
 
+# Options
+
+This is a summary of the supported options when instantiating a new client, and their default values:
+
+```ruby
+Bitreserve::Client.new(
+  # bearer_token for OAuth authentication
+  token: ENV['BITRESERVE_AUTH_TOKEN']
+
+  # use the sandbox api (https://api-sandbox.bitreserve.org) instead of the real one
+  sandbox: false
```

Comment:
Also, not having the endpoint as module config would make it really hard for someone to build a fake server, for instance. Because it would not control the creation of the client, but would be able to change configs at boot time. That's something else we are assuming here, that whoever is using this will have control of the entire process.